packages/comms/src/__tests__/https.ts

```diff
@@ -0,0 +1,64 @@
+import * as https from "https";
+import { ESPConnection } from "../espConnection";
+import { WorkunitsService } from "../services/wsWorkunits";
+import { trustwave } from "../pem/trustwave";
+
+const extraRootAgent = new https.Agent({
+    ca: trustwave
+});
+
+const acceptUnothorisedAgent = new https.Agent({
+    rejectUnauthorized: false
+});
+
+describe("https", () => {
+    it("fetch fail", () => {
+        return fetch("https://play.hpccsystems.com:18010/WsWorkunits/WUQuery.json").then(response => {
```

Comment:
Shouldn't this return false in the `.then` callback?

Reply:
Actually it should throw an error - as this is testing a case that is known to fail.

Reply:
The test is checking that it will fail without any cert of reject override - so its correct.